contrib/opencensus-ext-django/opencensus/ext/django/middleware.py

```diff
@@ -267,3 +272,29 @@ def process_response(self, request, response):
             log.error('Failed to trace request', exc_info=True)
         finally:
             return response
+
+    def process_exception(self, request, exception):
+        # Do not trace if the url is blacklisted
+        if utils.disable_tracing_url(request.path, self.blacklist_paths):
+            return
+
+        try:
+            if hasattr(exception, '__traceback__'):
+                tb = exception.__traceback__
+            else:
+                _, _, tb = sys.exc_info()
+
+            span = _get_django_span()
+            span.add_attribute(
+                attribute_key=ERROR_NAME,
+                attribute_value=exception.__class__.__name__)
+            span.add_attribute(
+                attribute_key=ERROR_MESSAGE,
+                attribute_value=str(exception))
+            span.add_attribute(
+                attribute_key=STACKTRACE,
+                attribute_value=traceback.format_tb(tb))
+
+            _set_django_attributes(span, request)
```

Comment:
Curious if `process_response` is always run even if there is an exception, so we might not need to call this?

Reply:
It depends where the exception is coming from, so no. It could be beforehand, in another middleware, anywhere really